tests/json_infra/conftest.py

```diff
@@ -243,15 +234,70 @@ def pytest_sessionstart(session: Session) -> None:  # noqa: U100
                 )
 
 
+def pytest_sessionstart(session: Session) -> None:  # noqa: U100
+    lock_path = session.config.rootpath.joinpath("tests/fixtures/.lock")
+
+    # use portalocker for mutmut runs
+    if os.environ.get("MUTANT_UNDER_TEST"):
+        shared_lock = portalocker.Lock(lock_path, flags=portalocker.LOCK_SH)
+        shared_lock.acquire()
```

Comment:
What happens if an exception is thrown while this lock is held? I assume the pytest process eventually exits?

If we aren't sure, we should probably use a try-except that unlocks and unsets the `mutmut_shared_lock`.